### PR TITLE
Fix retirement flash message

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/infrastructure/test_retirement.py -v -k 'unset' --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit e94d08a3236d2cf965533781aaa1024487578420:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav 5.4 failed due to appliance request not fulfilled. I think this should be OK to merge.

File: cfme/common/vm.py [hunk 1]
```diff
@@ -472,8 +472,12 @@ def retire(self):
         self.load_details(refresh=True)
         lcl_btn(self.TO_RETIRE, invokes_alert=True)
         sel.handle_alert()
-        flash.assert_success_message(
-            "Retire initiated for 1 VM and Instance from the CFME Database")
```
Comment (psav) [inline cfme/common/vm.py]: Personally I would prefer that we version pick the string, that way it's easier to add more if the versions change in future, and it's what pick was designed for.

Comment (mfalesni) [inline cfme/common/vm.py]: Ook!

**Comment by dajoRH** [comment]
Lint report for commit c766f66e54481e16677356cbe007cca4deadaee5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/common/vm.py (hunk 1 above)
Comment (mfalesni) [inline cfme/common/vm.py]: @psav I created a decorator that resolves a verpick dictionary if passed.

**Comment by mfalesni** [comment]
@psav Mergy mergy?

**Pull request merged**